bucket/rclone-browser.json

```
@@ -0,0 +1,55 @@
+{
+    "version": "1.8.0",
+    "description": "A simple GUI for rclone",
+    "homepage": "https://github.com/kapitainsky/RcloneBrowser",
+    "license": "MIT",
+    "depends": "rclone",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/kapitainsky/RcloneBrowser/releases/download/1.8.0/rclone-browser-1.8.0-a0b66c6-windows-64-bit.exe",
+            "hash": "sha512:1d465a5417f57ed3c203464ccebf3badef7217ad2bb8766ad5c08827082dd6ed1a4e9b3cdfa12ba71d53e0846ea4f3d123b18bfc414952c909676daca0442a0b"
+        },
+        "32bit": {
+            "url": "https://github.com/kapitainsky/RcloneBrowser/releases/download/1.8.0/rclone-browser-1.8.0-a0b66c6-windows-32-bit.exe",
+            "hash": "sha512:fdce63684ba687ae30e9f25c39fbbc7f875c49667eedeaaa9e59a25a1febcff928465a293c81721f648dce8569241a2a938ae16d5462cdd2c366a1ba13c1ee1b"
+        }
+    },
+    "innosetup": true,
+    "pre_install": [
+        "if (Test-Path \"$persist_dir\\RcloneBrowser.ini\") {",
+        "   Copy-Item \"$persist_dir\\RcloneBrowser.ini\" \"$dir\"",
+        "} else {",
+        "   Set-Content \"$dir\\RcloneBrowser.ini\" @('[Settings]', 'checkRcloneBrowserUpdates=false', 'checkRcloneUpdates=false') -Encoding Ascii",
+        "}",
+        "if (!(Test-Path \"$persist_dir\\tasks.bin\")) {",
+        "   New-Item \"$dir\\tasks.bin\" | Out-Null",
+        "}"
+    ],
+    "uninstaller": {
+        "script": "Copy-Item \"$dir\\RcloneBrowser.ini\" \"$persist_dir\" -ErrorAction SilentlyContinue -Force"
+    },
+    "shortcuts": [
+        [
+            "RcloneBrowser.exe",
+            "Rclone Browser"
+        ]
+    ],
+    "persist": "tasks.bin",
```

Does it use QT's atomic saving for tasks.bin file?

```suggestion
    "persist": [
        "downloads",
        "tasks.bin"
    ],
```

---

Not sure what it is but the tasks.bin file isn't recreated every time it's saved unlike the .ini file.